apps/dashboard/src/lib/components/Org/Community/index.svelte

```diff
@@ -85,7 +85,7 @@
   $: filteredDiscussions = discussions.filter(
     (discussion) =>
       discussion.title.toLowerCase().includes(searchValue.toLowerCase()) &&
-      (!selectedId || discussion.courseId === selectedId)
+      (!selectedId || selectedId === '' || discussion.courseId === selectedId)
```

Comment:
@tunny17 What's the purpose of the extra condition `selectedId === ''`?

Isn't it the same as `!selectedId`?